config/standard/.travis

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -eu
+
+if find test/ -name \*.js | xargs git grep -F .only; then
+	echo 'ERROR mocha .only() calls found in test files.  These must be removed!'
+	exit 1
+fi
+
+npm install
+npm test
```

Comment:
This seems weird to me - why is this `.` prefixed? It makes it look like a `.travis.yml` file...

We already have a grunt task to check for the `.only`s - why not expand that to check config too?

Then this script becomes so trivial it may as well be inlined into the .travis.yml I think.